### PR TITLE
FPS Logo

### PR DESCRIPTION
**Pull request by LaNeraNura**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/fps.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...iRERERMqXApaIiLSKT9/zwkiwvwCVO36umWjyDpBl 682/92Tnz+/8fe99IH8c8SLCSfMPF1t7Yr90hzuN7/ARj/m+f36wkwWstcDrDssMlrv76xC9htki j1j0+KUnrCzmzD0RERERKS8KWCIi0uJO/+8Xj7DI/krwAzGqdvx84miyw+cVsJp9sB2/q+QCluOr DF43eD3Aa+b2uuPLCSyLovB6U65i+YwrPr5FW46IiIiI5KOAJSIirepTf36+r1nFflEq7BsCBxi+ rxPt584BZr4v7vsBXfI+gAJW849bGgFrHdgb7qw2wgqc1Wa2MsCKyH1pLu2v98y+8/rEy4Y3aEsQ ERERkd2hgCUiIu1u2J3P9rHK9D4hRP1DxD5R8P4hYh/D+hG8H3g/h/5A1/d9owJWc9+1uwFrI/gK YLVbtAr3lQarPYQVFvnqkEutShGtWLOu5+q5mYGNWntFREREpC0oYImISIdx1h/n99zSZcs+FtJ9 nLC3hWhvPHyAyPu4295m9CHQF6Mvzt6ON39mV+cKWFvYen+p1eCrzX0tZmtx1rr7KvNobc7CmnTa V71N1cpZlxy9WWuaiIiIiJQaBSwRESlbQ+rm9ujamK3OBaoDXh051bjv6VDtUB29+3+g2qA6uPXC vdqgyqE7eDfY7h5e7R+w6sHfMXjLA2uIbC34Wg++2szWAGvB15qHteR8VXqPHmsf+eIRm7QmiIiI iEhHp4AlIiJSwLB7ntwj1PdM56LN1Z6tSKcs19MtdPEo1dVD6OFQEZlVE+JfV3Pv/Sns/IIcbP3W P/jGCJpC8EYs2hjl/C0P4e29+vZ4Z3StLtkTERERERERERERERERERERERERERERERERERERERER ERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERER EREREREREREREfmn/w+BS718bzxb3wAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOC0xMS0wOFQwMzoy MTo0OCswMDowML9tozIAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTgtMTEtMDhUMDM6MjE6NDgrMDA6 MDDOMBuOAAAAAElFTkSuQmCC" id="image844" opacity="1" stop-opacity="1"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/fps.svg:1]: - Can we remove the `<g id="g866">`? It it not used.
- Please remove all unused `id` attributes except the `<title id="">`
- Can we rename the `pi-{fps}` to `pi-fps`? It should only contains alphanumeric characters.

Comment (LaNeraNura) [reply app/assets/images/payment_icons/fps.svg]: Hi,
kindly help to verify the submitted changes. 

Thanks!

File: app/assets/images/payment_icons/fps.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/fps.svg:1]: 1. The `<title id="pi-fps">{FPS}</title>` should be `<title id="pi-fps">FPS</title>`
2. Please remove the `</g>` at the end. It is causing an error to render the svg.

**Comment by LaNeraNura** [comment]
Okay, the requested changes have been made.

**Pull request merged**